index.js

```diff
@@ -264,6 +264,16 @@ function attributesToString(attributes) {
     } else if (key === 'style') {
       result += ' style="' + styleToString(value) + '"';
       continue;
+    } else if (key === 'attrs') {
+      const regexAttributes = /\s*(\S+)=["']([^"']+)["']|\s*(\S+)=([^&\s]+)\s*/g
+      let resultAttrs = '';
+      let match;
+      while ((match = regexAttributes.exec(value)) !== null) {
```

Comment:
We could support `object | string`, when a string is used we do not escape it and directly insert into the tag, when a object is used, we can escape it and attach it, this way custom attributes may or may not be escaped.